### PR TITLE
Add a document showing how to simulate multi-data center scenarios

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve?

Add a document showing how to simulate multi-data center scenarios

### CODE REVIEW

File: website/docs/use_cases/multi_data_center.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+> **Note**: 
+> 
+> Currently, Chaos Mesh can't simulate the scenario of the bandwidth limitations between data centers. So in this case, 
+> only simulate the scenario of the latency between different data centers.
+
+## Experiment environment
+
+Suppose our application will be deployed in three data centers in a production environment 
+adn these data centers are still under construction. Now we want to test the impact of
```
Comment (AngleNet) [inline website/docs/use_cases/multi_data_center.md:22]: s/adn/and ?

**Comment by AngleNet** [comment]
@fewdan Could you resolve the conflicts ?

File: website/docs/use_cases/multi_data_center.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
+---
+id: multi_data_center 
+title: Simulate multi-data center
```
Comment (AngleNet) [inline website/docs/use_cases/multi_data_center.md:3]: "Network latency simulation across multiple data centers" could be better ?

**Comment by cwen0** [comment]
> @fewdan Could you resolve the conflicts ?

Thanks, I will fix it ASAP.

**Pull request merged**